sklearn/ensemble/voting_classifier.py

```diff
@@ -252,17 +269,29 @@ def transform(self, X):
         else:
             return self._predict(X)
 
+    def set_params(self, **params):
+        """ Setting the parameters for the voting classifier
+
+        Valid parameter keys can be listed with get_params().
+
+        Parameters
+        ----------
+        params: keyword arguments
+            Specific parameters using e.g. set_params(parameter_name=new_value)
```

Comment:
In addition, to setting the parameters of the ``VotingClassifier``, (doubleticks) the individual classifiers of the ``VotingClassifier`` can also be set or replaced by setting them to None.